locations/spiders/loves.py

```diff
@@ -26,23 +30,40 @@ def start_requests(self):
             }
         )
         yield scrapy.Request(
-            "https://www.loves.com/api/sitecore/StoreSearch/SearchStores",
+            f"https://www.loves.com/api/sitecore/StoreSearch/SearchStoresWithDetail?pageNumber={0}&top=50&lat=39.09574818760951&lng=-96.9935195",
             method="POST",
             body=payload,
             headers=HEADERS,
         )
 
     def parse(self, response):
         stores = response.json()
-        for store in stores[0]["Points"]:
-            yield Feature(
-                name=store["Name"],
-                ref=store["SiteId"],
-                street_address=store["Address1"],
-                city=store["City"],
-                state=store["State"],
-                postcode=store["Zip"],
-                phone=store["PhoneNumber"],
-                lat=float(store["Latitude"]),
-                lon=float(store["Longitude"]),
-            )
+        if len(stores) == 0:
+            raise CloseSpider()
+        else:
```

Comment:
There's no need for this `else` here. Can you remove it and un-indent the for loop?